private/pkg/git/gittest/gittest.go

```diff
@@ -164,11 +201,16 @@ func runInDir(t *testing.T, runner command.Runner, dir string, cmd string, args
 		command.RunWithStderr(stderr),
 	)
 	if err != nil {
-		t.Logf("run %q", strings.Join(append([]string{cmd}, args...), " "))
-		_, err := io.Copy(os.Stderr, stderr)
-		require.NoError(t, err)
+		require.FailNow(
```

Comment:
This improves the error message so that it's right in the test failures as opposed to stderr.
```
--- FAIL: TestCommitsToSyncWithNoPreviousSyncPoints (0.28s)
    gittest.go:204: 
                Error Trace:    /Users/saquib/dev/buf/private/pkg/git/gittest/gittest.go:204
                                                        /Users/saquib/dev/buf/private/pkg/git/gittest/repository.go:68
                                                        /Users/saquib/dev/buf/private/buf/bufsync/commits_to_sync_test.go:131
                                                        /Users/saquib/dev/buf/private/buf/bufsync/commits_to_sync_test.go:39
                Error:          git command failed: "git checkout bar": exit status 1
                Test:           TestCommitsToSyncWithNoPreviousSyncPoints
                Messages:       error: pathspec 'bar' did not match any file(s) known to git
```